specification/web/resource-manager/Microsoft.Web/stable/2016-09-01/AppServiceEnvironments.json

```diff
@@ -1924,13 +1924,13 @@
           "description": "Name/identifier of the diagnostics.",
           "type": "string"
         },
-        "diagnosicsOutput": {
+        "diagnosticsOutput": {
```

Comment:
@nschonni - Any reason to make a breaking change in an already deployed api-version? Have you already deployed this change? This will make it increasingly difficult for customers to using Azure stack. I would expect you to not touch the old api version and make a breaking change in  the new  api version (which is the definition of a breaking change).

Reply:
What happens if customers would send a request with old property names for api version 2016-09-01? Does the service alias both the property names on it's side?

Reply:
No idea how the previous versions work or if there is a redirect functionality available in the API to forward the requests to the other endpoint. I think I'm going to move the "environment" fix to a separate PR since that part is straight forward.

Reply:
@naveedaz - Can you take a look at this?

Reply:
Fixing typo stuff on the documentation that lead me up to this repo through the sdk ones.
If you want to file bugs to fix this in the next version, that's fine with me.